app/src/main/java/com/github/amusinamaria/ui/details/DetailsFragment.kt

```diff
@@ -45,7 +53,7 @@ class DetailsFragment : Fragment() {
 
     private fun updateCastData() {
         castAdapter.apply {
-            bindCastCards(MockDataSource().getCastCards())
+            bindCastCards(movieCard.cast)
```

Comment:
Do you really need to use APPLY for one function call?

Reply:
Done